### PR TITLE
fix: overflow Text in select #147

### LINKED ISSUE #147: Micronutrient data options' text overflows
Text may be longer than the width of the selection box.  How best to support this?  Text wrapping an option?

### PR DESCRIPTION
**Pull request by JohannDascalescu**
- Removed limit from select width
- Closes #147

### CODE REVIEW

File: src/app/pages/quickMaps/components/sideNavContent/sideNavContent.component.scss
Comment (jon571) [inline src/app/pages/quickMaps/components/sideNavContent/sideNavContent.component.scss:161]: Don't use ::ng-deep at root or it will affect everything, everywhere.
Use :host ::ng-deep to at least keep it within the component.

Also the sub-selectors need "&" on the start :-)

**Review by jon571 — COMMENTED** [review]
Just a couple of quick changes needed for the css file. :-)

**Comment by jon571** [comment]
I've re-worked this and just removed the width restriction on the popout list.

The wrapping worked but the unlimited width is better.

I removed the tooltip as it was inconsistent with the other selects.  If a user wants to see the full non-ellipsis version they can click on the select again. :-)

**Comment by Fuhji** [comment]
@bgsandan waiting for your review/approval still?

**Review by bgsandan — APPROVED** [review]
Lets hit the big green button™

**Pull request merged**
